### PR TITLE
Added deno starters

### PR DESCRIPTION
**Pull request by Meldiron**
Deno versions:

- deno-1.12
- deno-1.13
- deno-1.14

### CODE REVIEW

File: deno-1.12/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## 🤖 Documentation
+
+Simple function similar to typical "hello world" example, but instead, we return a simple JSON that tells everyone how awesome developers are.
+
+<!-- Update with your description, for example 'Create Stripe payment and return payment URL' -->
+
+_Example input:_
+
+No input
```
Comment (christyjacob4) [inline deno-1.12/README.md:15]:
```suggestion
This function expects no input 
```

File: deno-1.12/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+```json
+{
+ "areDevelopersAwesome": true
+}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud functions:
```
Comment (christyjacob4) [inline deno-1.12/README.md:31]:
```suggestion
List of environment variables used by this cloud function:
```

File: deno-1.12/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+- **APPWRITE_FUNCTION_ENDPOINT** - Endpoint of Appwrite project
+- **APPWRITE_FUNCTION_API_KEY** - Appwrite API Key
+<!-- Add your custom environments variables -->
```
Comment (christyjacob4) [inline deno-1.12/README.md:35]:
```suggestion
<!-- Add your custom environment variables -->
```

File: deno-1.12/README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+## 🚀 Deployment
+
+There are two ways of deploying the Appwrite function, both having the same results, but each using a different process. We highly recommend using CLI deployment to achieve the best experience.
+
+### Using CLI
+
+Make sure you have [Appwrite CLI](https://appwrite.io/docs/command-line#installation) installed, and you have successfully logged into your Appwrite server. To make sure AppwriteCLI is ready, you can use the command `appwrite client --debug` and it should respond with green text `✓ Success`.
```
Comment (christyjacob4) [inline deno-1.12/README.md:43]:
```suggestion
Make sure you have [Appwrite CLI](https://appwrite.io/docs/command-line#installation) installed, and you have successfully logged into your Appwrite server. To make sure Appwrite CLI is ready, you can use the command `appwrite client --debug` and it should respond with green text `✓ Success`.
```

File: deno-1.12/README.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+Make sure you are in the same folder as your `appwrite.json` file and run `appwrite deploy function` to deploy your function. You will be prompted to select which functions you want to deploy.
+
+### Manual using tar.gz
+
+Manual deployment has no requirements and uses Appwrite Console to deploy the tag. First, enter the folder of your function. Then, create a tarball of the whole folder and gzip it. After creating `.tar.gz` file, visit Appwrite Console, click on the `Deploy Tag` button and switch to the `Manual` tab. There, set the `entrypoint` to `src/mod.ts`, and upload the file we just generated. Finally, it's considered good practice to remove the build file from your function source folder.
```
Comment (christyjacob4) [inline deno-1.12/README.md:49]:
```suggestion
Manual deployment has no requirements and uses Appwrite Console to deploy the tag. First, enter the folder of your function. Then, create a tarball of the whole folder and gzip it. After creating `.tar.gz` file, visit Appwrite Console, click on the `Deploy Tag` button and switch to the `Manual` tab. There, set the `entrypoint` to `src/mod.ts`, and upload the file we just generated.
```

File: deno-1.12/src/mod.ts [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+let teams = new sdk.Teams(client);
+let users = new sdk.Users(client);
+
+client
+    .setEndpoint(Deno.env.get('APPWRITE_FUNCTION_ENDPOINT') as string)
+    .setProject(Deno.env.get('APPWRITE_FUNCTION_PROJECT_ID') as string)
+    .setKey(Deno.env.get('APPWRITE_FUNCTION_API_KEY') as string);
+
+if (!Deno.env.get('APPWRITE_FUNCTION_ENDPOINT') || !Deno.env.get('APPWRITE_FUNCTION_API_KEY')) {
+    console.warn("Some environment variables are not set. Function cannot use AppwriteSDK properly.");
```
Comment (christyjacob4) [inline deno-1.12/src/mod.ts:22]:
```suggestion
    console.warn("Some environment variables are not set. Function cannot use Appwrite SDK properly.");
```

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Some minor changes. Please propagate this to all examples.

**Comment by Meldiron** [comment]
Also make sure to fix spelling:
```
let helath = new sdk.Health(client); 
```

EDIT: Resolved.

**Pull request closed without merging**